tests/appsec/test_shell_execution.py

```diff
@@ -69,16 +72,15 @@ def setup_truncated(self):
     )
     def test_truncated(self):
         shell_exec_span = self.fetch_command_execution_span(self.r_truncation)
+        assert shell_exec_span["resource"] == "echo"
```

Comment:
already a problem in the code before, but the local var (shell_exec_span) should be renamed

Reply:
Changing it to just `span`, I don't think we need that much disambiguation in such a short scope.